src/jobs/bullmq.ts

```diff
@@ -133,9 +137,25 @@ export async function runWorkers(queueNames: QueueName[]) {
   logger.info({ queueNames }, `Workers started successfully`);
 }
 
+export async function closeAllQueues() {
+  await Promise.all(
+    [...Queues.values()].map(async (queue) => {
+      if (!(await queue.isPaused())) {
```

Comment:
Why pause? I'd expect `queue.closing`.

Reply:
For workers, this is handling the case when a worker has been created but not started yet. For queues, I just did it to be symmetrical.

If you don't call `runWorkers` the `worker.close` fails to close an unopened connection. (this should be handled in bullmq, but it's not)